usr/tgtd.c

```diff
@@ -925,7 +979,19 @@ static int target_delete(const _ha_request *reqp,
 			"tgt cmd too long");
 		return HA_CALLBACK_CONTINUE;
 	}
```

Comment:
Where is the else part ? In else there  should be a command without "force" option.

Reply:
Already present Sir.